sympy/printing/smtlib.py

```diff
@@ -42,6 +44,12 @@ class SMTLibPrinter(Printer):
             StrictLessThan: '<',
             StrictGreaterThan: '>',
 
+            Q.eq: '=',
```

Comment:
I think these lines might be what's causing the sphinx errors. I think I might need to import these without using Q.